health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MapChart.js

```
@@ -0,0 +1,196 @@
+                    onMouseLeave={(event) => onMouseLeave(geo, current, event)}
+                  />
+                );
+              })
+            }
+          </Geographies>
+        </ComposableMap>
+        <span className="map-status">
+          {STATUS.map((sta) => {
+            return (
+              <span className="map-row">
+                <span className="map-box" style={{ background: getColor({ status: sta }) }}></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)